src/main/java/commands/EventCommand.java

```diff
@@ -14,7 +14,7 @@
 public class EventCommand extends Command {
 
     @Override
-    public void run(Bot bot, String[] args) {
+    public String[] run(Bot bot, String[] args) {
         if (args[0].equals("")) {
             throw new InvalidArgumentsException(Ui.ERROR_SIGNATURE + "The description of an event cannot be empty.");
         }
```

Comment:
This two seems like mutually exclusive events, perhaps you could use if else to connect them? 
For example, you could create the splitArgs array at the start. Then if splitArgs[0].equals(""), give first message, else if splitArgs.length < 2, give second message. 

I personally recommend avoid single if statements recurring in one function, as you may missed out certain logically flaws that the current if condition could not catch, but could lead to runtime exceptions.

Reply:
Point well taken! In this case, I don't think there are any functional differences but combining the two definitely makes the mutual exclusivity more obvious.